tests/interactive_broker_version.py

```diff
@@ -100,12 +100,36 @@ def test_version(version, cmd=None, deploy=True, conf={}, debug=False,
                 break
         elif mech == 'OAUTHBEARER':
             security_protocol = 'SASL_PLAINTEXT'
-            os.write(
-                fd, ('enable.sasl.oauthbearer.unsecure.jwt=true\n'.encode(
-                    'ascii')))
-            os.write(fd, ('sasl.oauthbearer.config=%s\n' %
-                          'scope=requiredScope principal=admin').encode(
-                              'ascii'))
+            if defconf.get('oidc_method') == 'OIDC':
+                os.write(
+                    fd, ('sasl.oauthbearer.method=OIDC\n'.encode(
+                         'ascii')))
+                os.write(
+                    fd, ('sasl.oauthbearer.client.id=123\n'.encode(
+                         'ascii')))
+                os.write(
+                    fd, ('sasl.oauthbearer.client.secret=abc\n'.encode(
+                         'ascii')))
+                os.write(
+                    fd, ('sasl.oauthbearer.extensions=\
+                         ExtensionworkloadIdentity=develC348S,\
+                         Extensioncluster=lkc123\n'.encode(
+                         'ascii')))
+                os.write(
+                    fd, ('sasl.oauthbearer.scope=test\n'.encode(
+                         'ascii')))
+                cmd_env['VALID_OIDC_URL'] = "http://localhost:8080/retrieve"
```

Comment:
I tried this way, but since trivup part is not merged yet, I met the build issue from CI, in my local it works fine. 
```
./interactive_broker_version.py -c "make quick" 2.7.0) || travis_terminate 1 ; fi
Traceback (most recent call last):
  File "./interactive_broker_version.py", line 14, in <module>
    from trivup.apps.OauthbearerOIDCApp import OauthbearerOIDCApp
ImportError: No module named 'trivup.apps.OauthbearerOIDCApp'
```
Can we keep this way for current? cmd_env['VALID_OIDC_URL'] = "http://localhost:8080/retrieve"
Or waiting for CI to merge.